### ISSUE TITLE
Errors running `pod lib create`

### ISSUE THREAD

**Issue by stevendesu** [CocoaPods/CocoaPods#8932]
* [x] I've read and understood the [*CONTRIBUTING* guidelines and have done my best effort to follow](https://github.com/CocoaPods/CocoaPods/blob/master/CONTRIBUTING.md).

# Report

## What did you do?

Run `pod lib create test`

## What did you expect to happen?

Create a new pod and open the pod in Xcode, with no errors

## What happened instead?

Several errors were thrown, and Xcode did not open

## Specific Output

```
Stevens-MacBook-Pro:cocoapods stevenbarnett$ pod lib create test2
Cloning `https://github.com/CocoaPods/pod-template.git` into `test2`.
Configuring test2 template.

------------------------------

To get you started we need to ask a few questions, this should only take a minute.

If this is your first time we recommend running through with the guide: 
 - https://guides.cocoapods.org/making/using-pod-lib-create.html
 ( hold cmd and double click links to open in a browser. )


What platform do you want to use?? [ iOS / macOS ]
 > 
ios
What language do you want to use?? [ Swift / ObjC ]
 > ObjC

Would you like to include a demo application with your library? [ Yes / No ]
 > 
yes
Which testing frameworks will you use? [ Specta / Kiwi / None ]
 > 
specta
Would you like to do view based testing? [ Yes / No ]
 > 
yes
What is your class prefix?
 > test

Running pod install on your new library.

Analyzing dependencies
Fetching podspec for `test2` from `../`
Downloading dependencies
Installing Expecta (1.0.6)
Installing Expecta+Snapshots (3.1.1)
Installing FBSnapshotTestCase (2.1.4)
Installing Specta (1.0.7)
Installing test2 (0.1.0)
[!] Unable to determine Swift version for the following pods:

- `FBSnapshotTestCase` does not specify a Swift version and none of the targets (`test2_Tests`) integrating it have the `SWIFT_VERSION` attribute set. Please contact the author or set the `SWIFT_VERSION` attribute in at least one of the targets that integrate this pod.

 Ace! you're ready to go!
 We will start you off by opening your project in Xcode
  open 'test2/Example/test2.xcworkspace'
The file /Users/stevenbarnett/Source/experiments/cocoapods/test2/Example/test2.xcworkspace does not exist.

To learn more about the template see `https://github.com/CocoaPods/pod-template.git`.
To learn more about creating a new pod, see `https://guides.cocoapods.org/making/making-a-cocoapod`.
```

Two issues to note in the outupt:

 - `FBSnapshotTestCase` does not specify a Swift version and none of the targets (`test2_Tests`) integrating it have the `SWIFT_VERSION` attribute set.
 - The file /Users/stevenbarnett/Source/experiments/cocoapods/test2/Example/test2.xcworkspace does not exist.

### Stack

```
   CocoaPods : 1.7.2
        Ruby : ruby 2.3.7p456 (2018-03-28 revision 63024) [universal.x86_64-darwin18]
    RubyGems : 2.5.2.3
        Host : Mac OS X 10.14.5 (18F203)
       Xcode : 10.2.1 (10E1001)
         Git : git version 2.20.1 (Apple Git-117)
Ruby lib dir : /System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib
Repositories : master - https://github.com/CocoaPods/Specs.git @ bb0654e6fe24e5a171d728ed3430d2f727c6f843
```

### Installation Source

```
Executable Path: /usr/local/bin/pod
```

### Plugins

```
cocoapods-deintegrate : 1.0.4
cocoapods-plugins     : 1.0.0
cocoapods-search      : 1.0.0
cocoapods-stats       : 1.1.0
cocoapods-trunk       : 1.3.1
cocoapods-try         : 1.1.0
```

## Additional Details

I tried changing several of the parameters that I used during library initialization to see the results:

 - Changing `iOS` to `MacOS`: I was not prompted for a language (forced to use Swift) but there were no errors and Xcode opened
 - Changing `ObjC` to `Swift`: There were no errors and Xcode opened
 - Changing `Yes` to `No` for "would you like to use a demo application": Errors presented as described above
 - Changing `Specta` to `Kiwi`: Errors presented as described above
 - Changing `Specta` to `None`: Errors presented as described above
 - Changing `Yes` to `No` for "would you like to do view-based testing": There were no errors and Xcode opened
 - Changing the class prefix to anything other than blank: Errors presented as described above
 - Changing the class prefix to blank: "You need to provide an answer"

In short, the issue seems to be specifically related to doing view-based testing in Objective-C

**Comment by dnkoutso** [comment]
Will have to verify. Most likely a bug.

**Comment by dnkoutso** [comment]
Confirmed.

**Comment by dnkoutso** [comment]
PR is out. https://github.com/CocoaPods/pod-template/pull/236 

No need for release of CocoaPods, re running `pod lib create` should clone the latest template.
